client/client.go

```diff
@@ -259,6 +265,26 @@ func (c *Client) Pay(channelId types.Destination, amount *big.Int) {
 	c.engine.PaymentRequestsFromAPI <- engine.PaymentRequest{ChannelId: channelId, Amount: amount}
 }
 
+// CreatePayment returns (via a go channel) a voucher that increases the redeemable balance
+// of the given payment channel by the given amount.
+//
+// The returned voucher must be sent to the payee.
+func (c *Client) CreatePayment(channelId types.Destination, amount *big.Int) (<-chan payments.Voucher, error) {
```

Comment:
Somewhat out of scope: are we happy with sprinkling "payment" widely through our API and codebase? "Voucher" has fewer connotations and we get to define what it means. But I feel like "payment" has the opportunity to mislead, given the replace-by-incentive nature of vouchers?

The godoc you have here is good, I think elsewhere existing godoc is not as clear about what calling `Pay` actually means.

Reply:
A good question that has two alternative answers. The case for removing `Payment` verbiage is as you described - a new terminology carries less baggage.

On the other hand, pay and payment are universally understood. Vouchers are implementation details that we could consider obscuring from the API entirely.

Agreed on the godoc, although we may suffer the curse of too much knowledge - an outsider wouldn't look at a function like `Pay(amount)` and wonder whether this function bumps total spend up _to_ `amount` or _by_ `amount`. (I did wonder this, and dug through code to find out). Obvious regular usage is that it bumps total spend _by_ amount.